cli/cmd/tapRunner.go

```diff
@@ -109,11 +109,13 @@ func RunMizuTap(podRegexQuery *regexp.Regexp, tappingOptions *MizuTapOptions) {
 }
 
 func readValidationRules(file string) (string, error) {
-	content, err := ioutil.ReadFile(file)
+	// content, err := ioutil.ReadFile(file)
```

Comment:
Remove?

Reply:
I'm using the shared.Decode... to validate the file and get the final valid yaml string.

Reply:
I meant the commented out code.